src/index.js

```diff
@@ -0,0 +1,30 @@
+import './style.scss';
+import Weather from './weather';
+import displayInfo from './displayInfo';
+
+const searchForm = document.querySelector('form');
+const page = document.querySelector('body');
+const header = document.querySelector('header');
+
+
+searchForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const cityName = document.querySelector('#city-name').value;
+
+  if (cityName === '') {
+    alert('please enter a valid city');
```

Comment:
### [Optional]

It would be nice if you could remove any `alert` and use of `window()` in your codebase;

This should be production code, we shouldn't have `alert`, please remove the `alert`.

You can make use of a `div` instead of `alert`

![image](https://user-images.githubusercontent.com/31897434/101089622-b3fd3280-35b5-11eb-8c55-7a8e8173a049.png)

https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635172/

@lekegitrepo kindly sign to this too 😄

Reply:
A Custom alert component has been created and has replaced the previous Window.alert()

Reply:
> ### [Optional]
> It would be nice if you could remove any `alert` and use of `window()` in your codebase;
> 
> This should be production code, we shouldn't have `alert`, please remove the `alert`.
> 
> You can make use of a `div` instead of `alert`
> 
> ![image](https://user-images.githubusercontent.com/31897434/101089622-b3fd3280-35b5-11eb-8c55-7a8e8173a049.png)
> 
> https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635172/
> 
> @lekegitrepo kindly sign to this too

**_Using `alert` is not allowed as mentioned by @bellom. It is marked as optional here because I mistakenly miss it :disappointed:  when I was reviewing the code._**